### ISSUE TITLE
Improve visibility detection.

### ISSUE THREAD

**Issue by vrde** [tracking-exposed/web-extension#42]
In order to extract the visibility of a post, we analyze the last word in the "aria-label" of the post itself. This part should be improved because there might be some really weird corner cases, for example if someone's first name is one of the `publicWords` (see `/src/scrape.js`), then we might detect a wrong visibility.

**Comment by vecna** [comment]
Considering #82, we might detect when 0 posts are collected, and ask the adopter to provide a sample of what is seeing/assist them in this

**Comment by vecna** [comment]
The first step would be addressed in the next milestone, still, no UX would be provided to get any assistance from the adopter.

**Comment by vecna** [comment]
UX is provided and server side collection of anomalies is implemented
